COBOL Programming Course #4 - Testing/COBOL Programming Course #4 - Testing.md

```diff
@@ -329,4 +329,36 @@ The difficulty in achieving this has resulted in falling back to manual processe
 
 ## How can test automation be achieved on z/OS?
 
-Many organizations have built, or are now building, CI/CD pipelines for their z/OS applications, with considerable success.  These teams are looking to test automation to help them achieve an efficient pipeline. This chapter described an approach to test automation on z/OS, by using a framework for automating tests which offers deep integration with z/OS capabilities.
\ No newline at end of file
+Many organizations have built, or are now building, CI/CD pipelines for their z/OS applications, with considerable success.  These teams are looking to test automation to help them achieve an efficient pipeline. This chapter described an approach to test automation on z/OS, by using a framework for automating tests which offers deep integration with z/OS capabilities.
+
+# Cobol Check
+
+In this chapter, we discuss the concept of unit testing in cobol-programming language with the use of Cobol Check. It delivers precise, fine-grained unit testing/checking capabilities for Cobol, matching the conceptual level of detail found in unit testing frameworks designed for popular languages like Python, Ruby, C#, and Java.
+
+
+- **Introduction to Cobol Check**
+
+  - **What is Cobol Check ?**
+  - **Where Can you run Cobol Check ?**
+  - **How does Cobol Check accomplish unit testing?**
```

Comment:
Can you help to remove the space between Check and ?

And also replace Cobol Check with COBOL Check